src/main/java/de/zalando/zmon/scheduler/ng/entities/EntityRepository.java

```diff
@@ -24,6 +25,9 @@
 @Component
 public class EntityRepository extends CachedRepository<String, EntityAdapterRegistry, Entity> {
 
+    @Autowired
+    Scheduler scheduler;
```

Comment:
Best practice is to use constructor injection over field or setter injection (e.g. https://www.vojtechruzicka.com/field-dependency-injection-considered-harmful/)